src/game/utils/mapObjects.ts

```diff
@@ -0,0 +1,59 @@
+import * as fs from 'fs'
+import * as path from 'path'
+
+export interface RawBoundingBox {
+    min: [number, number, number]
+    max: [number, number, number]
+}
+
+export interface RawMapObject {
+    name: string
+    boundingBox: RawBoundingBox
+}
+
+export interface BoundingBox {
+    min: { x: number; y: number; z: number }
+    max: { x: number; y: number; z: number }
+}
+
+export interface MapObject {
+    name: string
+    boundingBox: BoundingBox
+}
+
+// 파일 경로 설정
+const mapObjectsFilePath = path.join(__dirname, '../data/mapObjects.json')
+
+// 파일 읽기 및 파싱
+const mapObjectsData = fs.readFileSync(mapObjectsFilePath, 'utf-8')
+const mapObjects = JSON.parse(mapObjectsData)
```

Comment:
```suggestion
// data/mapObjects.ts
export const mapObjects: RawMapObject[] = [
    {
        name: 'house1_color_0',
        boundingBox: {
            min: [-232.120849609375, -0.9097098681438661, -46.38691393874643],
            max: [-109.00450134277344, 74.13478343813486, 103.86822561653825],
        },
    },
    ...
]
// 사용처
import { mapObjects } from '..'
```

json 파일은 관리가 편하지만, I/O read가 필요해서 객체로 import 하면 어떨까요? 
type check도 되구, 제안드려봅니다

Reply:
객체로 import하면 메모리에 다 올리는건가 싶어서 파일로 뺀거긴한데 생각해보니 너무 자주 읽게 될거같긴하네